content/inference/particlenet.md

```diff
@@ -1088,69 +1195,76 @@ Here we present three ways of training. For readers who have a local machine wit
         python train.py \
-        --data-train '<path-to-samples>/prep/top_train_*.awkd' \
-        --data-val '<path-to-samples>/prep/top_val_*.awkd' \
-        --fetch-by-file --fetch-step 1 \
-        --num-workers 3 \
-        --data-config data/benchmark/${data_config} \
-        --network-config networks/benchmark/${model_config} \
-        --model-prefix output/${prefix} \
-        --gpus 0 --batch-size 1024 --start-lr 5e-3 --num-epochs 20 --optimizer ranger \
-        --log output/${prefix}.train.log
-
-        # Predicting score
+         --data-train ${PATH_TO_SAMPLES}'/prep/top_train_*.root' \
+         --data-val ${PATH_TO_SAMPLES}'/prep/top_val_*.root' \
+         --fetch-by-file --fetch-step 1 --num-workers 3 \
+         --data-config top_tagging/data/${DATA_CONFIG} \
+         --network-config top_tagging/networks/${MODEL_CONFIG} \
+         --model-prefix output/${PREFIX} \
+         --gpus 0 --batch-size 1024 --start-lr 5e-3 --num-epochs 20 --optimizer ranger \
+         --log output/${PREFIX}.train.log
+
+        # Predicting score, using 1 GPU
         python train.py --predict \
-        --data-test '<path-to-samples>/prep/top_test_*.awkd' \
-        --num-workers 3 \
-        --data-config data/benchmark/${data_config} \
-        --network-config networks/benchmark/${model_config} \
-        --model-prefix output/${prefix}_best_epoch_state.pt \
-        --gpus 0 --batch-size 1024 \
-        --predict-output output/${prefix}_predict.root
+         --data-test ${PATH_TO_SAMPLES}'/prep/top_test_*.root' \
+         --num-workers 3 \
+         --data-config top_tagging/data/${DATA_CONFIG} \
+         --network-config top_tagging/networks/${MODEL_CONFIG} \
+         --model-prefix output/${PREFIX}_best_epoch_state.pt \
+         --gpus 0 --batch-size 1024 \
+         --predict-output output/${PREFIX}_predict.root
+
+        [ -d "runs/" ] && tar -caf output.tar output/ runs/ || tar -caf output.tar output/
         ```
 
     ???+ hint "HTCondor submitted file: `submit.sub`"
-        ```linenums="1"
+        Modify the argument line. These are the bash variable `PREFIX`, `MODEL_CONFIG`, `DATA_CONFIG`, `PATH_TO_SAMPLES` used in the `Weaver` command. Since the EOS directory is accessable accross all condor nodes on lxplus, one may directly specify `<your-path-to-samples>` as the EOS path provided above. An example is shown in the commented line.
+        ```bash linenums="1" hl_lines="3"
         Universe                = vanilla
         executable              = run.sh
-        arguments               = 
-        output                  = logs/$(ClusterId).$(ProcId).out
-        error                   = logs/$(ClusterId).$(ProcId).err
-        log                     = logs/$(ClusterId).log
+        arguments               = <prefix> <model-config> <data-config> <your-path-to-samples>
+        #arguments              = mlp mlp_pf.py pf_features.yaml /eos/user/c/coli/public/weaver-benchmark/top_tagging/samples
```

Comment:
Why do you have a line here which is commented out?

Reply:
Here we are trying to provide a real configuration example, so people will not doubt whether to add quote marks etc in a condor configuration line. This is mentioned above: "An example is shown in the commented line."